src/layout/Summary/SummaryBoilerplate.module.css

```diff
@@ -0,0 +1,13 @@
+.container {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-between;
+}
+.label {
+  font-weight: 500;
+}
```

Comment:
Is the label supposed to look that thicc? It used to have a weight of 400:

![image](https://user-images.githubusercontent.com/47412359/232723937-9215e063-d021-4fa8-bbdc-ecd350c0b931.png)
![image](https://user-images.githubusercontent.com/47412359/232724029-f6237bc6-05cc-40b0-b7b9-f356b75e5865.png)

Reply:
There is a lot of **bold** text now

![image](https://user-images.githubusercontent.com/47412359/232724753-ef22f08d-f1ff-49ca-85de-b5517fb304ad.png)

Reply:
Hm yeah, that didn't look that good. Will fix it, thx!